### PR TITLE
Cirrus CI setup

### PR DESCRIPTION
**Pull request by seaki**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+container:
+  dockerfile: Dockerfile
```
Comment (fkorotkov) [inline .cirrus.yml]: A container from `Dockerfile` will be pre-build and **cached** so it won't catch changes to your `Gemfile.lock`. Consider creating a separate Dockerfile for CI or maybe add `rspec` to your Dockerfile and enjoy [Zero-configuration Docker builds](https://medium.com/cirruslabs/zero-config-docker-builds-with-cirrus-ci-703db9c1e04e).

Comment (seaki) [reply .cirrus.yml]: Okay, I created separate Dockerfile and it seems to work. Thanks.

Comment (fkorotkov) [reply .cirrus.yml]: Glad it worked for you! 🙌

Comment (seaki) [reply .cirrus.yml]: @fkorotkov It seems not to working, how can I set up Zero-configuration Docker builds for separate Dockerfile for CI?

Comment (fkorotkov) [reply .cirrus.yml]: Fixed here https://github.com/seaki/rubicure-graphql/pull/37

**Pull request merged**